### PR TITLE
Updating 36 vulnerabilities

### PR DESCRIPTION
**Pull request by talos-vulndev**
Updating 36 vulnerabilities

### CODE REVIEW

**Comment by cve-team** [comment]
talos-vulndev is not recognized currently as an account participating in the Git Pilot. To ensure timely processing of your pull request, please contact the CVE Team using either the CVE Request web form (https://cveform.mitre.org/) or an email (cve@mitre.org) and confirm your identity.

File: 2022/21xxx/CVE-2022-21145.json [hunk 1]
```diff
@@ -1,18 +1,76 @@
 {
```
Comment (cve-team) [inline 2022/21xxx/CVE-2022-21145.json:1]: Invalid source.

File: 2022/21xxx/CVE-2022-21154.json [hunk 2]
```diff
@@ -1,18 +1,76 @@
 {
```
Comment (cve-team) [inline 2022/21xxx/CVE-2022-21154.json:1]: Invalid source.

File: 2022/21xxx/CVE-2022-21210.json [hunk 3]
```diff
@@ -1,18 +1,76 @@
 {
```
Comment (cve-team) [inline 2022/21xxx/CVE-2022-21210.json:1]: Invalid source.

File: 2022/21xxx/CVE-2022-21234.json [hunk 4]
```diff
@@ -1,18 +1,76 @@
 {
```
Comment (cve-team) [inline 2022/21xxx/CVE-2022-21234.json:1]: Invalid source.

File: 2022/22xxx/CVE-2022-22149.json [hunk 5]
```diff
@@ -1,18 +1,76 @@
 {
```
Comment (cve-team) [inline 2022/22xxx/CVE-2022-22149.json:1]: Invalid source.

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

File: 2022/21xxx/CVE-2022-21145.json [hunk 6]
```diff
@@ -1,18 +1,76 @@
...
+    },
+    "references": {
+        "reference_data": [
+            {
+                "refsource": "MISC",
+                "name": "https://talosintelligence.com/vulnerability_reports/TALOS-2022-1442",
+                "url": "https://talosintelligence.com/vulnerability_reports/TALOS-2022-1442"
+            },
+            {
+                "refsource": "https://www.lansweeper.com/changelog/",
```
Comment (anthnysingleton) [inline 2022/21xxx/CVE-2022-21145.json]: This should not be an URL. Needs to be set to "CONFIRM" or "MISC".

File: 2022/21xxx/CVE-2022-21154.json [hunk 7]
```diff
@@ -1,18 +1,76 @@
...
+    },
+    "references": {
+        "reference_data": [
+            {
+                "refsource": "MISC",
+                "name": "https://talosintelligence.com/vulnerability_reports/TALOS-2022-1464",
+                "url": "https://talosintelligence.com/vulnerability_reports/TALOS-2022-1464"
+            },
+            {
+                "refsource": "https://files.leadtools.com/index.php/s/joFz7BcCZYMot5Q",
```
Comment (anthnysingleton) [inline 2022/21xxx/CVE-2022-21154.json]: This should not be an URL. Needs to be set to "CONFIRM" or "MISC".

File: 2022/21xxx/CVE-2022-21210.json [hunk 8]
```diff
@@ -1,18 +1,76 @@
...
+    },
+    "references": {
+        "reference_data": [
+            {
+                "refsource": "MISC",
+                "name": "https://talosintelligence.com/vulnerability_reports/TALOS-2022-1444",
+                "url": "https://talosintelligence.com/vulnerability_reports/TALOS-2022-1444"
+            },
+            {
+                "refsource": "https://www.lansweeper.com/changelog/",
```
Comment (anthnysingleton) [inline 2022/21xxx/CVE-2022-21210.json]: This should not be an URL. Needs to be set to "CONFIRM" or "MISC".

File: 2022/21xxx/CVE-2022-21234.json [hunk 9]
```diff
@@ -1,18 +1,76 @@
...
+    },
+    "references": {
+        "reference_data": [
+            {
+                "refsource": "MISC",
+                "name": "https://talosintelligence.com/vulnerability_reports/TALOS-2022-1443",
+                "url": "https://talosintelligence.com/vulnerability_reports/TALOS-2022-1443"
+            },
+            {
+                "refsource": "https://www.lansweeper.com/changelog/",
```
Comment (anthnysingleton) [inline 2022/21xxx/CVE-2022-21234.json]: This should not be an URL. Needs to be set to "CONFIRM" or "MISC".

File: 2022/22xxx/CVE-2022-22149.json [hunk 10]
```diff
@@ -1,18 +1,76 @@
...
+    },
+    "references": {
+        "reference_data": [
+            {
+                "refsource": "MISC",
+                "name": "https://talosintelligence.com/vulnerability_reports/TALOS-2022-1441",
+                "url": "https://talosintelligence.com/vulnerability_reports/TALOS-2022-1441"
+            },
+            {
+                "refsource": "https://www.lansweeper.com/changelog/",
```
Comment (anthnysingleton) [inline 2022/22xxx/CVE-2022-22149.json]: This should not be an URL. Needs to be set to "CONFIRM" or "MISC".

**Comment by talos-vulndev** [comment]
Should be fixed now.

**Pull request merged**